### PR TITLE
Add post widget

### PR DESCRIPTION
**Pull request by hht-hannah**
Add an initial version of post widget. Add List view.
Screenshot_20200825_235348_com example Graminsta

### CODE REVIEW

**Comment by shiyi001** [comment]
We should add a `Follow` button on the right of `username`?

File: flutter/Graminsta/lib/post/post_widget.dart [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import 'package:Graminsta/models/post.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+Widget postWidget(Post post) {
+  String _markedUser = '';
+  String _mentionUser = '';
+  for (String username in post.markedUser) {
+    _markedUser += username + ", ";
+  }
```
Comment (Time1ess) [inline flutter/Graminsta/lib/post/post_widget.dart]: Can be simplified into

```
final markedUser = post.markedUser.join(', ');
```

File: flutter/Graminsta/lib/post/post_widget.dart [hunk 2]
```diff
@@ -0,0 +1,122 @@
+import 'package:Graminsta/config.dart';
+import 'package:Graminsta/models/post.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+Widget postWidget(Post post) {
+  String _markedUser = '';
```
Comment (Time1ess) [inline flutter/Graminsta/lib/post/post_widget.dart]: No need for the underscore, as this is a local variable and can't be accessed from outside this function.

**Review by Time1ess — APPROVED** [review]

**Review by ArianaGu — APPROVED** [review]

**Comment by hht-hannah** [comment]
> We should add a `Follow` button on the right of `username`?

Sure, I will add the Follow button in another PR.

**Pull request merged**